Stratis.Bitcoin/Builder/IFullNode.cs

```diff
@@ -10,5 +12,8 @@ public interface IFullNode : IDisposable
         System.Version Version { get; }
 
         void Start();
+        Task RunAsync();
```

Comment:
Maybe this can go in an extension class instead of on the interface (similar to WebHostExtensions maybe FullNodeHostExtensions)